src/components/DataTable.tsx

```diff
@@ -161,8 +161,8 @@ export function DataTable<T extends object>(props: Props<T>) {
 
   const { data, isLoading, isError } = useQuery(
     [
+      apiEndpoint.pathname,
       props.endpoint,
```

Comment:
@beganovich This change is just in purpose of testing this PR, because the same change is on the previous PR. So, when we merge that PR I will resolve conflicts here. But, this is just for David's testing.